index.rst

```diff
@@ -28,7 +28,7 @@ Data Preview 0 (DP0) is the first of three data previews during the period leadi
 The goals of DP0 are to serve as an early integration test of the Legacy Survey of Space and Time (LSST) Science Pipelines and the Rubin Science Platform (RSP), and to enable a limited number of astronomers and students to begin early preparations for science with the LSST.
 
 .. Important::
-    **New to DP0?** Welcome! See the :ref:`Delegate-Homepage-Getting-Started-Checklist`.
+    To access DP0 and become a delegate, see the :ref:`Delegate-Homepage-Getting-Started-Checklist`.
```

Comment:
"To become a delegate and access the DP0"

Reply:
incorporated -- that is better wording thank you!